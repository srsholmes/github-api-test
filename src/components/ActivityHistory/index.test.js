import {render, fireEvent, waitForElement} from '@testing-library/react';

import {ActivityHistory} from './index';

jest.mock('axios');

describe('Test 1', () => {
  it('Check A', async () => {
    const actual = true;
    expect(actual).toBe(false);
  });
});

