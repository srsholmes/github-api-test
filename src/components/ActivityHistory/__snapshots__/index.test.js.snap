// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityHistory displays the activities with the contributions 1`] = `
<div>
  <h4
    class="title"
  >
    Commit activity:
  </h4>
  <ul
    class="activityWrapper"
  >
    <li
      class="repo"
    >
      <p
        class="repoName"
      >
        <a
          href="http://github.com/undefined/progit2"
          target="_blank"
        >
          progit2
        </a>
      </p>
      <p>
        Recent Commit History Dates:
      </p>
      <ul
        class="commitsWrapper"
      >
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-06-01&to=2019-06-30"
            target="_blank"
          >
            Wed Jun 19 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-06-01&to=2019-06-30"
            target="_blank"
          >
            Fri Jun 14 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-06-01&to=2019-06-30"
            target="_blank"
          >
            Sun Jun 02 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-06-01&to=2019-06-30"
            target="_blank"
          >
            Sat Jun 01 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-05-01&to=2019-05-31"
            target="_blank"
          >
            Mon May 13 2019
          </a>
        </li>
      </ul>
    </li>
    <li
      class="repo"
    >
      <p
        class="repoName"
      >
        <a
          href="http://github.com/undefined/progit2-bg"
          target="_blank"
        >
          progit2-bg
        </a>
      </p>
      <p>
        Recent Commit History Dates:
      </p>
      <ul
        class="commitsWrapper"
      >
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-05-01&to=2019-05-31"
            target="_blank"
          >
            Mon May 13 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-04-01&to=2019-04-30"
            target="_blank"
          >
            Mon Apr 15 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-04-01&to=2019-04-30"
            target="_blank"
          >
            Wed Apr 10 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-04-01&to=2019-04-30"
            target="_blank"
          >
            Thu Apr 04 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-03-01&to=2019-03-31"
            target="_blank"
          >
            Sat Mar 30 2019
          </a>
        </li>
      </ul>
    </li>
    <li
      class="repo"
    >
      <p
        class="repoName"
      >
        <a
          href="http://github.com/undefined/ben.github.com"
          target="_blank"
        >
          ben.github.com
        </a>
      </p>
      <p>
        Recent Commit History Dates:
      </p>
      <ul
        class="commitsWrapper"
      >
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-03-01&to=2019-03-31"
            target="_blank"
          >
            Mon Mar 11 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-03-01&to=2019-03-31"
            target="_blank"
          >
            Sun Mar 10 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-03-01&to=2019-03-31"
            target="_blank"
          >
            Sat Mar 09 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2019-02-01&to=2019-02-28"
            target="_blank"
          >
            Thu Feb 21 2019
          </a>
        </li>
        <li
          class="commitTime"
        >
          <a
            href="https://github.com/ben?tab=overview&from=2018-12-01&to=2018-12-31"
            target="_blank"
          >
            Wed Dec 12 2018
          </a>
        </li>
      </ul>
    </li>
  </ul>
</div>
`;
